src/main.c

```diff
@@ -2481,13 +2498,13 @@ void handle_movement(double dt) {
         }
         s->x += vx;
         s->y += vy + dy * ut;
-        s->z += vz;
-        if (collide(2, &s->x, &s->y, &s->z)) {
+        s->z += vz;   		// collide - height is the block height or character height  
```

Comment:
are these LSEP characters intended? Doesnt look like a line break intended here

Reply:
that block is used to test the collision and movements